test/unit/api-client/api-client.submit.spec.ts

```diff
@@ -143,6 +155,59 @@ describe('api-client job submission', () => {
         });
     });
 
+    describe('submitJob', () => {
+        it('submit job with legacy options', async () => {
+            const options: RevAiJobOptions = {
+                metadata: null,
+                media_url: mediaUrl,
+                callback_url: callbackUrl,
+                custom_vocabulary_id: null,
+                custom_vocabularies: null,
+                skip_punctuation: null,
+                skip_diarization: null,
+                speaker_channels_count: null,
+                filter_profanity: null,
+                remove_disfluencies: null,
+                delete_after_seconds: null,
+                language: null,
+                transcriber: null
```

Comment:
Do these need to be assigned `null`? Since they typed to be `<*> | undefined` because of `?`, I don't think these need to be assigned, could be wrong